report/lambda_function.py

```diff
@@ -302,7 +374,9 @@ def plot_table(df_releases: DataFrame, rows: int) -> Chart:
         alt.Text("value", type="nominal"),
     ).properties(
         width=1300,
+        height=200
     )
+    chart.save("chart.png")
```

Comment:
Is this save just a test that hasn't been removed or do you later open the file?